apps/css-workshop/src/pages/table.astro

```diff
@@ -2750,84 +2734,77 @@ import Checkbox_ from '../components/Checkbox.astro';
           </Button_>
 
           <span class='iui-table-paginator-pages-group'>
-            <button
-              class='iui-button iui-field iui-table-paginator-page-button'
-              data-iui-variant='borderless'
-              data-iui-size='small'
+            <Button_
+              class='iui-table-paginator-page-button'
+              variant='borderless'
+              size='small'
               aria-label='Page 1'
-              type='button'
             >
               <span>1</span>
-            </button>
+            </Button_>
 
             <span class='iui-table-paginator-ellipsis iui-table-paginator-ellipsis-small'
               >&mldr;</span
             >
 
-            <button
-              class='iui-button iui-field iui-table-paginator-page-button'
-              data-iui-variant='borderless'
-              data-iui-size='small'
+            <Button_
+              class='iui-table-paginator-page-button'
+              variant='borderless'
+              size='small'
               aria-label='Page 13'
-              type='button'
             >
               <span>13</span>
-            </button>
+            </Button_>
 
-            <button
-              class='iui-button iui-field iui-table-paginator-page-button'
-              data-iui-variant='borderless'
-              data-iui-size='small'
+            <Button_
+              class='iui-table-paginator-page-button'
+              variant='borderless'
+              size='small'
               aria-label='Page 14'
-              type='button'
             >
               <span>14</span>
-            </button>
+            </Button_>
 
-            <button
-              class='iui-button iui-field iui-table-paginator-page-button'
-              data-iui-variant='borderless'
-              data-iui-size='small'
-              data-iui-active='true'
+            <Button_
+              class='iui-table-paginator-page-button'
+              variant='borderless'
+              size='small'
               aria-label='Page 15'
-              type='button'
+              data-iui-active='true'
```

Comment:
You can use the `active` prop instead.

```suggestion
              active={true}
```

Same comment for the similar places.